src/app/src/sass/05_base/_base.scss

```diff
@@ -25,7 +25,7 @@ body {
     bottom: 0;
     left: 0;
     width: 100%;
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
I think this CSS property is making the screen too wide. I didn't try it out, but be explicit about `overflow-x` and `overflow-y`?
web:
<img width="818" alt="screen shot 2019-02-04 at 10 05 14 am" src="https://user-images.githubusercontent.com/10568752/52216835-2ccdbf00-2865-11e9-9201-c4621a8e0e67.png">

ipad pro:
<img width="628" alt="screen shot 2019-02-04 at 10 13 46 am" src="https://user-images.githubusercontent.com/10568752/52217109-c5fcd580-2865-11e9-83de-abfea243dd47.png">
<img width="653" alt="screen shot 2019-02-04 at 10 13 58 am" src="https://user-images.githubusercontent.com/10568752/52217110-c5fcd580-2865-11e9-9e90-c754a69e05a4.png">

Reply:
Good catch. I'm not seeing that on my laptop, but I'm going to test it out on my iPad since that's where this will be deployed.